openlibrary/plugins/upstream/borrow.py

```diff
@@ -198,6 +198,15 @@ def POST(self, key):  # noqa: PLR0915
             lending.s3_loan_api(s3_keys, ocaid=edition.ocaid, action='leave_waitlist')
             stats.increment('ol.loans.leaveWaitlist')
             raise web.redirect(edition_redirect)
+        elif action == 'locate':
+            redirect_url = 'https://search.worldcat.org/'
+            if edition.get('oclc_numbers', None):
+                redirect_url += f'title/{edition.oclc_numbers[0]}'
+            elif edition.get('isbn_13', edition.get('isbn_10', None)):
+                redirect_url += f'isbn/{edition.isbn_13[0] or edition.isbn_10[0]}'
```

Comment:
This is problematic because in line 205 we check whether isbn13 or isbn10 is available... But consider the case where there is an isbn10 but not an isbn13

In line 206, even though there's no isbn13, we're still trying to access the 0th element which doesn't exist.

Reply:
Oh, good catch!